openlibrary/templates/diff.html

```diff
@@ -33,9 +33,9 @@ <h2 class="collapse"><a href="$a.key">$title</a> &rarr; Diff</h2>
         <a href="$changequery({}, v=page.revision)"><span class="uppercase"><strong>$_("Revision %d", page.revision)</strong></span></a>
         $ v = get_version(page.key, page.revision)
         $if v.author:
-            <span class="brown">$:_('by <a href="%s">%s</a> %s', v.author.key, v.author.displayname, datestr(v.created))</span>
+            <span class="brown">$:_('by') <a href="$v.author.key">$v.author.displayname</a> $datestr(v.created)</span>
         $else:
-            <span class="brown">$:_('by Anonymous %s', datestr(v.created))</span>
+            <span class="brown">$:_('by Anonymous') $datestr(v.created)</span>
```

Comment:
Do we not want the date string converted (independently)? I see that we specifically want 'by Anonymous' matched (and that the date may complicate and prevent this from matching) but do we still want to attempt to $_ the date independently?

Also, is $_ sufficient here? Or do we need to render using $:_.

Reply:
![screenshot from 2018-02-22 14 53 37](https://user-images.githubusercontent.com/905545/36516111-678c577c-17e1-11e8-9e4b-0c63d5829d75.png)


`datestr()` does the translation based on `lang=` itself, so that is already happening.
I think `$:_` is overkill on the `by Anonymous` as there is nothing that needs escaping anyway.  Other `$datestr()` don't have it, see those on the About part of the main page:
https://github.com/internetarchive/openlibrary/blob/708ce7a34198e3f6135139bce22fa9b1585ab856/openlibrary/templates/home/about.html#L32